tests/console/wavpack.pm

```diff
@@ -16,12 +16,23 @@ use warnings;
 use testapi;
 use utils;
 use version_utils 'is_sle';
-use registration 'add_suseconnect_product';
+use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
 
 sub run {
     # setup
     select_console 'root-console';
-    add_suseconnect_product('sle-module-desktop-applications') if is_sle("15-sp1+");
+    # development module needed for dependencies, released products are tested with sdk module
+    if (get_var('BETA')) {
+        my $sdk_repo = is_sle('15+') ? get_var('REPO_SLE_MODULE_DEVELOPMENT_TOOLS') : get_var('REPO_SLE_SDK');
+        zypper_ar 'http://' . get_var('OPENQA_URL') . "/assets/repo/$sdk_repo", "SDK";
```

Comment:
this breaks ability to clone this job to any private openQA instance

Reply:
Well, to be correct, when you sync the complete repos to your local openQA instance the tests should still work :)